src/Local_Development/Extras.php

```diff
@@ -105,6 +105,21 @@ public function page_init() {
 				]
 			);
 		}
+
+		if( in_array( $_SERVER['REMOTE_ADDR'], [ '127.0.0.1', '::1' ] ) ) {
```

Comment:
in_array needs third parameter `true` for WPCS

Reply:
Ok thanks, unfortunately my local php config is temporarily KO ...